e2e/tests/121-Portal/MakeNewPayment/MakeSuccessfulPaymentOCW.spec.ts

```diff
@@ -0,0 +1,124 @@
+import LoginPage from '@121-e2e/pages/Login/LoginPage';
+import RegistrationDetails from '@121-e2e/pages/RegistrationDetails/RegistrationDetailsPage';
+import TableModule from '@121-e2e/pages/Table/TableModule';
+import NLRCProgram from '@121-service/seed-data/program/program-nlrc-ocw.json';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { registrationsOCW } from '@121-service/test/registrations/pagination/pagination-data';
+import { expect, test } from '@playwright/test';
+import englishTranslations from '../../../../interfaces/Portal/src/assets/i18n/en.json';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const programIdOCW = 3;
+  const OcwProgramId = programIdOCW;
+
+  const accessToken = await getAccessToken();
+  await seedIncludedRegistrations(registrationsOCW, OcwProgramId, accessToken);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/login');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[28274] Make Successful payment for OCW', async ({ page }) => {
+  const table = new TableModule(page);
+  const homePage = new HomePage(page);
+  const registration = new RegistrationDetails(page);
+
+  const numberOfPas = registrationsOCW.length;
+  const defaultTransferValue = NLRCProgram.fixedTransferValue;
+  const maxTransferValue = registrationsOCW.reduce((output, pa) => {
+    return output + pa.paymentAmountMultiplier * defaultTransferValue;
+  }, 0);
+
+  await test.step('Navigate to PA table', async () => {
+    await homePage.navigateToProgramme(NLRCProgram.titlePortal.en);
+  });
+
+  await test.step('Initiate Do payment #1', async () => {
+    const paymentNr = 1;
+    const doPaymentLabel = englishTranslations.page.program[
+      'program-people-affected'
+    ].actions.doPayment.replace('{{paymentNr}}', paymentNr.toString());
+
+    await table.applyBulkAction({ label: doPaymentLabel });
+  });
+
+  await test.step('Verify that the do payment popup contains the right values', async () => {
+    const paIncludedLabel = englishTranslations.page.program['program-payout'][
+      'make-payment'
+    ]['number-included'].replace('{{number}}', numberOfPas.toString());
+    const maximumAmountLabel = englishTranslations.page.program[
+      'program-payout'
+    ]['total-amount'].replace(
+      '{{totalCost}}',
+      `EUR ${maxTransferValue.toString()}`,
+    );
+
+    await expect(page.getByText(paIncludedLabel)).toBeVisible();
+    await expect(page.getByText(maximumAmountLabel)).toBeVisible();
+    await expect(
+      registration.paymentPopupMakePaymentAmountInput.locator('input'),
+    ).toHaveValue(defaultTransferValue.toString());
```

Comment:
Can maybe some of it be transferred to one of the POMs and like tableModule and become eg. validateCorrectPaymentValues?